### ISSUE TITLE
[Bug]: i686 clang is broken when compiling code

### ISSUE THREAD

**Issue by truboxl** [termux/termux-packages#7938]
### Problem description

clang and clang++ (or ld.lld) seems to be missing `-L /system/lib` in i686 arch

### What steps will reproduce the bug?

Tested using https://github.com/termux/termux-docker on WSL2

```
~ $ echo "int main(void) { return 0; }" > hello.c
~ $ gcc hello.c -o hello
ld.lld: error: unable to find library -ldl
ld.lld: error: unable to find library -lc
ld.lld: error: unable to find library -ldl
clang-13: error: linker command failed with exit code 1 (use -v to see invocation)
~ $ gcc hello.c -o hello -v
clang version 13.0.0
Target: i686-unknown-linux-android24
Thread model: posix
InstalledDir: /data/data/com.termux/files/usr/bin
 "/data/data/com.termux/files/usr/bin/clang-13" -cc1 -triple i686-unknown-linux-android24 -emit-obj -mrelax-all --mrelax-relocations -mnoexecstack -disable-free -disable-llvm-verifier -discard-value-names -main-file-name hello.c -mrelocation-model pic -pic-level 2 -pic-is-pie -mframe-pointer=all -fno-rounding-math -mconstructor-aliases -target-cpu i686 -target-feature +ssse3 -tune-cpu generic -debugger-tuning=gdb -v -fcoverage-compilation-dir=/data/data/com.termux/files/home -resource-dir /data/data/com.termux/files/usr/lib/clang/13.0.0 -isysroot /data/data/com.termux/files -internal-isystem /data/data/com.termux/files/usr/lib/clang/13.0.0/include -internal-isystem /data/data/com.termux/files/usr/local/include -internal-externc-isystem /data/data/com.termux/files/usr/include/i686-linux-android -internal-externc-isystem /data/data/com.termux/files/include -internal-externc-isystem /data/data/com.termux/files/usr/include -fdebug-compilation-dir=/data/data/com.termux/files/home -ferror-limit 19 -fgnuc-version=4.2.1 -fcolor-diagnostics -o /data/data/com.termux/files/usr/tmp/hello-57ddf9.o -x c hello.c
clang -cc1 version 13.0.0 based upon LLVM 13.0.0 default target i686-unknown-linux-android24
ignoring nonexistent directory "/data/data/com.termux/files/usr/local/include"
ignoring nonexistent directory "/data/data/com.termux/files/include"
#include "..." search starts here:
#include <...> search starts here:
 /data/data/com.termux/files/usr/lib/clang/13.0.0/include
 /data/data/com.termux/files/usr/include/i686-linux-android
 /data/data/com.termux/files/usr/include
End of search list.
 "/data/data/com.termux/files/usr/bin/ld.lld" --sysroot=/data/data/com.termux/files -pie -z noexecstack --warn-shared-textrel -z now -z relro --hash-style=gnu --enable-new-dtags -rpath=/data/data/com.termux/files/usr/lib --eh-frame-hdr -m elf_i386 -dynamic-linker /system/bin/linker -o hello /data/data/com.termux/files/usr/lib/crtbegin_dynamic.o -L/data/data/com.termux/files/usr/lib /data/data/com.termux/files/usr/tmp/hello-57ddf9.o /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl -lc /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl /data/data/com.termux/files/usr/lib/crtend_android.o
ld.lld: error: unable to find library -ldl
ld.lld: error: unable to find library -lc
ld.lld: error: unable to find library -ldl
clang-13: error: linker command failed with exit code 1 (use -v to see invocation)
~ $ gcc hello.c -o hello -v -L /system/lib
clang version 13.0.0
Target: i686-unknown-linux-android24
Thread model: posix
InstalledDir: /data/data/com.termux/files/usr/bin
 "/data/data/com.termux/files/usr/bin/clang-13" -cc1 -triple i686-unknown-linux-android24 -emit-obj -mrelax-all --mrelax-relocations -mnoexecstack -disable-free -disable-llvm-verifier -discard-value-names -main-file-name hello.c -mrelocation-model pic -pic-level 2 -pic-is-pie -mframe-pointer=all -fno-rounding-math -mconstructor-aliases -target-cpu i686 -target-feature +ssse3 -tune-cpu generic -debugger-tuning=gdb -v -fcoverage-compilation-dir=/data/data/com.termux/files/home -resource-dir /data/data/com.termux/files/usr/lib/clang/13.0.0 -isysroot /data/data/com.termux/files -internal-isystem /data/data/com.termux/files/usr/lib/clang/13.0.0/include -internal-isystem /data/data/com.termux/files/usr/local/include -internal-externc-isystem /data/data/com.termux/files/usr/include/i686-linux-android -internal-externc-isystem /data/data/com.termux/files/include -internal-externc-isystem /data/data/com.termux/files/usr/include -fdebug-compilation-dir=/data/data/com.termux/files/home -ferror-limit 19 -fgnuc-version=4.2.1 -fcolor-diagnostics -o /data/data/com.termux/files/usr/tmp/hello-3ed08c.o -x c hello.c
clang -cc1 version 13.0.0 based upon LLVM 13.0.0 default target i686-unknown-linux-android24
ignoring nonexistent directory "/data/data/com.termux/files/usr/local/include"
ignoring nonexistent directory "/data/data/com.termux/files/include"
#include "..." search starts here:
#include <...> search starts here:
 /data/data/com.termux/files/usr/lib/clang/13.0.0/include
 /data/data/com.termux/files/usr/include/i686-linux-android
 /data/data/com.termux/files/usr/include
End of search list.
 "/data/data/com.termux/files/usr/bin/ld.lld" --sysroot=/data/data/com.termux/files -pie -z noexecstack --warn-shared-textrel -z now -z relro --hash-style=gnu --enable-new-dtags -rpath=/data/data/com.termux/files/usr/lib --eh-frame-hdr -m elf_i386 -dynamic-linker /system/bin/linker -o hello /data/data/com.termux/files/usr/lib/crtbegin_dynamic.o -L/system/lib -L/data/data/com.termux/files/usr/lib /data/data/com.termux/files/usr/tmp/hello-3ed08c.o /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl -lc /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl /data/data/com.termux/files/usr/lib/crtend_android.o
```
```
~ $ echo "int main(void) { return 0; }" > hello.cpp
~ $ g++ hello.cpp -o hello
ld.lld: error: unable to find library -ldl
ld.lld: error: unable to find library -lm
ld.lld: error: unable to find library -lc
ld.lld: error: unable to find library -ldl
clang-13: error: linker command failed with exit code 1 (use -v to see invocation)
~ $ g++ hello.cpp -o hello -v
clang version 13.0.0
Target: i686-unknown-linux-android24
Thread model: posix
InstalledDir: /data/data/com.termux/files/usr/bin
 "/data/data/com.termux/files/usr/bin/clang-13" -cc1 -triple i686-unknown-linux-android24 -emit-obj -mrelax-all --mrelax-relocations -mnoexecstack -disable-free -disable-llvm-verifier -discard-value-names -main-file-name hello.cpp -mrelocation-model pic -pic-level 2 -pic-is-pie -mframe-pointer=all -fno-rounding-math -mconstructor-aliases -target-cpu i686 -target-feature +ssse3 -tune-cpu generic -debugger-tuning=gdb -v -fcoverage-compilation-dir=/data/data/com.termux/files/home -resource-dir /data/data/com.termux/files/usr/lib/clang/13.0.0 -isysroot /data/data/com.termux/files -internal-isystem /data/data/com.termux/files/usr/include/c++/v1 -internal-isystem /data/data/com.termux/files/usr/lib/clang/13.0.0/include -internal-isystem /data/data/com.termux/files/usr/local/include -internal-externc-isystem /data/data/com.termux/files/usr/include/i686-linux-android -internal-externc-isystem /data/data/com.termux/files/include -internal-externc-isystem /data/data/com.termux/files/usr/include -fdeprecated-macro -fdebug-compilation-dir=/data/data/com.termux/files/home -ferror-limit 19 -fgnuc-version=4.2.1 -fcxx-exceptions -fexceptions -fcolor-diagnostics -D__GCC_HAVE_DWARF2_CFI_ASM=1 -o /data/data/com.termux/files/usr/tmp/hello-bd5c82.o -x c++ hello.cpp
clang -cc1 version 13.0.0 based upon LLVM 13.0.0 default target i686-unknown-linux-android24
ignoring nonexistent directory "/data/data/com.termux/files/usr/local/include"
ignoring nonexistent directory "/data/data/com.termux/files/include"
#include "..." search starts here:
#include <...> search starts here:
 /data/data/com.termux/files/usr/include/c++/v1
 /data/data/com.termux/files/usr/lib/clang/13.0.0/include
 /data/data/com.termux/files/usr/include/i686-linux-android
 /data/data/com.termux/files/usr/include
End of search list.
 "/data/data/com.termux/files/usr/bin/ld.lld" --sysroot=/data/data/com.termux/files -pie -z noexecstack --warn-shared-textrel -z now -z relro --hash-style=gnu --enable-new-dtags -rpath=/data/data/com.termux/files/usr/lib --eh-frame-hdr -m elf_i386 -dynamic-linker /system/bin/linker -o hello /data/data/com.termux/files/usr/lib/crtbegin_dynamic.o -L/data/data/com.termux/files/usr/lib /data/data/com.termux/files/usr/tmp/hello-bd5c82.o -lc++_shared /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl -lm -lc /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl /data/data/com.termux/files/usr/lib/crtend_android.o
ld.lld: error: unable to find library -ldl
ld.lld: error: unable to find library -lm
ld.lld: error: unable to find library -lc
ld.lld: error: unable to find library -ldl
clang-13: error: linker command failed with exit code 1 (use -v to see invocation)
~ $ g++ hello.cpp -o hello -v -L /system/lib
clang version 13.0.0
Target: i686-unknown-linux-android24
Thread model: posix
InstalledDir: /data/data/com.termux/files/usr/bin
 "/data/data/com.termux/files/usr/bin/clang-13" -cc1 -triple i686-unknown-linux-android24 -emit-obj -mrelax-all --mrelax-relocations -mnoexecstack -disable-free -disable-llvm-verifier -discard-value-names -main-file-name hello.cpp -mrelocation-model pic -pic-level 2 -pic-is-pie -mframe-pointer=all -fno-rounding-math -mconstructor-aliases -target-cpu i686 -target-feature +ssse3 -tune-cpu generic -debugger-tuning=gdb -v -fcoverage-compilation-dir=/data/data/com.termux/files/home -resource-dir /data/data/com.termux/files/usr/lib/clang/13.0.0 -isysroot /data/data/com.termux/files -internal-isystem /data/data/com.termux/files/usr/include/c++/v1 -internal-isystem /data/data/com.termux/files/usr/lib/clang/13.0.0/include -internal-isystem /data/data/com.termux/files/usr/local/include -internal-externc-isystem /data/data/com.termux/files/usr/include/i686-linux-android -internal-externc-isystem /data/data/com.termux/files/include -internal-externc-isystem /data/data/com.termux/files/usr/include -fdeprecated-macro -fdebug-compilation-dir=/data/data/com.termux/files/home -ferror-limit 19 -fgnuc-version=4.2.1 -fcxx-exceptions -fexceptions -fcolor-diagnostics -D__GCC_HAVE_DWARF2_CFI_ASM=1 -o /data/data/com.termux/files/usr/tmp/hello-edc694.o -x c++ hello.cpp
clang -cc1 version 13.0.0 based upon LLVM 13.0.0 default target i686-unknown-linux-android24
ignoring nonexistent directory "/data/data/com.termux/files/usr/local/include"
ignoring nonexistent directory "/data/data/com.termux/files/include"
#include "..." search starts here:
#include <...> search starts here:
 /data/data/com.termux/files/usr/include/c++/v1
 /data/data/com.termux/files/usr/lib/clang/13.0.0/include
 /data/data/com.termux/files/usr/include/i686-linux-android
 /data/data/com.termux/files/usr/include
End of search list.
 "/data/data/com.termux/files/usr/bin/ld.lld" --sysroot=/data/data/com.termux/files -pie -z noexecstack --warn-shared-textrel -z now -z relro --hash-style=gnu --enable-new-dtags -rpath=/data/data/com.termux/files/usr/lib --eh-frame-hdr -m elf_i386 -dynamic-linker /system/bin/linker -o hello /data/data/com.termux/files/usr/lib/crtbegin_dynamic.o -L/system/lib -L/data/data/com.termux/files/usr/lib /data/data/com.termux/files/usr/tmp/hello-edc694.o -lc++_shared /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl -lm -lc /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl /data/data/com.termux/files/usr/lib/crtend_android.o
```

Same issue when using `i686-linux-android-clang`
```
~ $ i686-linux-android-clang hello.c -o hello
ld.lld: error: unable to find library -ldl
ld.lld: error: unable to find library -lc
ld.lld: error: unable to find library -ldl
clang-13: error: linker command failed with exit code 1 (use -v to see invocation)
~ $ i686-linux-android-clang hello.c -o hello -v
clang version 13.0.0
Target: i686-unknown-linux-android
Thread model: posix
InstalledDir: /data/data/com.termux/files/usr/bin
 "/data/data/com.termux/files/usr/bin/clang-13" -cc1 -triple i686-unknown-linux-android -emit-obj -mrelax-all --mrelax-relocations -mnoexecstack -disable-free -disable-llvm-verifier -discard-value-names -main-file-name hello.c -mrelocation-model pic -pic-level 2 -fhalf-no-semantic-interposition -mframe-pointer=all -fno-rounding-math -mconstructor-aliases -target-cpu i686 -target-feature +ssse3 -tune-cpu generic -debugger-tuning=gdb -v -fcoverage-compilation-dir=/data/data/com.termux/files/home -resource-dir /data/data/com.termux/files/usr/lib/clang/13.0.0 -isysroot /data/data/com.termux/files -internal-isystem /data/data/com.termux/files/usr/lib/clang/13.0.0/include -internal-isystem /data/data/com.termux/files/usr/local/include -internal-externc-isystem /data/data/com.termux/files/usr/include/i686-linux-android -internal-externc-isystem /data/data/com.termux/files/include -internal-externc-isystem /data/data/com.termux/files/usr/include -fdebug-compilation-dir=/data/data/com.termux/files/home -ferror-limit 19 -fgnuc-version=4.2.1 -fcolor-diagnostics -o /data/data/com.termux/files/usr/tmp/hello-db028d.o -x c hello.c
clang -cc1 version 13.0.0 based upon LLVM 13.0.0 default target i686-unknown-linux-android24
ignoring nonexistent directory "/data/data/com.termux/files/usr/local/include"
ignoring nonexistent directory "/data/data/com.termux/files/include"
#include "..." search starts here:
#include <...> search starts here:
 /data/data/com.termux/files/usr/lib/clang/13.0.0/include
 /data/data/com.termux/files/usr/include/i686-linux-android
 /data/data/com.termux/files/usr/include
End of search list.
 "/data/data/com.termux/files/usr/bin/ld.lld" --sysroot=/data/data/com.termux/files -z noexecstack --warn-shared-textrel -z now -z relro --hash-style=both --enable-new-dtags -rpath=/data/data/com.termux/files/usr/lib --eh-frame-hdr -m elf_i386 -dynamic-linker /system/bin/linker -o hello /data/data/com.termux/files/usr/lib/crtbegin_dynamic.o -L/data/data/com.termux/files/usr/lib /data/data/com.termux/files/usr/tmp/hello-db028d.o /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl -lc /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl /data/data/com.termux/files/usr/lib/crtend_android.o
ld.lld: error: unable to find library -ldl
ld.lld: error: unable to find library -lc
ld.lld: error: unable to find library -ldl
clang-13: error: linker command failed with exit code 1 (use -v to see invocation)
~ $ i686-linux-android-clang hello.c -o hello -v -L /system/lib
clang version 13.0.0
Target: i686-unknown-linux-android
Thread model: posix
InstalledDir: /data/data/com.termux/files/usr/bin
 "/data/data/com.termux/files/usr/bin/clang-13" -cc1 -triple i686-unknown-linux-android -emit-obj -mrelax-all --mrelax-relocations -mnoexecstack -disable-free -disable-llvm-verifier -discard-value-names -main-file-name hello.c -mrelocation-model pic -pic-level 2 -fhalf-no-semantic-interposition -mframe-pointer=all -fno-rounding-math -mconstructor-aliases -target-cpu i686 -target-feature +ssse3 -tune-cpu generic -debugger-tuning=gdb -v -fcoverage-compilation-dir=/data/data/com.termux/files/home -resource-dir /data/data/com.termux/files/usr/lib/clang/13.0.0 -isysroot /data/data/com.termux/files -internal-isystem /data/data/com.termux/files/usr/lib/clang/13.0.0/include -internal-isystem /data/data/com.termux/files/usr/local/include -internal-externc-isystem /data/data/com.termux/files/usr/include/i686-linux-android -internal-externc-isystem /data/data/com.termux/files/include -internal-externc-isystem /data/data/com.termux/files/usr/include -fdebug-compilation-dir=/data/data/com.termux/files/home -ferror-limit 19 -fgnuc-version=4.2.1 -fcolor-diagnostics -o /data/data/com.termux/files/usr/tmp/hello-23b260.o -x c hello.c
clang -cc1 version 13.0.0 based upon LLVM 13.0.0 default target i686-unknown-linux-android24
ignoring nonexistent directory "/data/data/com.termux/files/usr/local/include"
ignoring nonexistent directory "/data/data/com.termux/files/include"
#include "..." search starts here:
#include <...> search starts here:
 /data/data/com.termux/files/usr/lib/clang/13.0.0/include
 /data/data/com.termux/files/usr/include/i686-linux-android
 /data/data/com.termux/files/usr/include
End of search list.
 "/data/data/com.termux/files/usr/bin/ld.lld" --sysroot=/data/data/com.termux/files -z noexecstack --warn-shared-textrel -z now -z relro --hash-style=both --enable-new-dtags -rpath=/data/data/com.termux/files/usr/lib --eh-frame-hdr -m elf_i386 -dynamic-linker /system/bin/linker -o hello /data/data/com.termux/files/usr/lib/crtbegin_dynamic.o -L/system/lib -L/data/data/com.termux/files/usr/lib /data/data/com.termux/files/usr/tmp/hello-23b260.o /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl -lc /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl /data/data/com.termux/files/usr/lib/crtend_android.o
```

### System information

termux-info:

```
/data/data/com.termux/files/usr/bin/getprop: 3: exec: /system/bin/getprop: not found
/data/data/com.termux/files/usr/bin/getprop: 3: exec: /system/bin/getprop: not found
/data/data/com.termux/files/usr/bin/getprop: 3: exec: /system/bin/getprop: not found
Application version:
unsupported
Packages CPU architecture:
i686
Subscribed repositories:
# sources.list
deb https://packages-cf.termux.org/apt/termux-main stable main
# game-repo (sources.list.d/game.list)
deb https://packages-cf.termux.org/apt/termux-games games stable
# science-repo (sources.list.d/science.list)
deb https://packages-cf.termux.org/apt/termux-science science stable
Updatable packages:
All packages up to date
Android version:

Kernel build information:
Linux 4473917a847b 5.10.60.1-microsoft-standard-WSL2 #1 SMP Wed Aug 25 23:20:18 UTC 2021 i686 Android
Device manufacturer:

Device model:

```

**Comment by buttaface** [comment]
[We patch the clang compiler to add that path if it exists](https://github.com/termux/termux-packages/blob/master/packages/libllvm/clang-lib-Driver-ToolChains-Linux.cpp.patch#L28), but looking into it now, [that `OSLibDir` variable was always wrongly set to `lib32` for Android i686](https://github.com/llvm/llvm-project/blob/main/clang/lib/Driver/ToolChains/Linux.cpp#L165).

My guess is that the gold linker added that `/system/lib` path back in, but since we switched to lld now with NDK 23, that needed path is no longer passed in anywhere, because lld doesn't hardcode such system paths and only uses what clang gives it.

I'll push a fix for i686.

**Comment by buttaface** [comment]
Should be fixed by latest `13.0.0-3` update, close after checking.

**Comment by truboxl** [comment]
Verified the update works. Closing...
```
~ $ clang hello.c -o hello -v
clang version 13.0.0
Target: i686-unknown-linux-android24
Thread model: posix
InstalledDir: /data/data/com.termux/files/usr/bin
 "/data/data/com.termux/files/usr/bin/clang-13" -cc1 -triple i686-unknown-linux-android24 -emit-obj -mrelax-all --mrelax-relocations -mnoexecstack -disable-free -disable-llvm-verifier -discard-value-names -main-file-name hello.c -mrelocation-model pic -pic-level 2 -pic-is-pie -mframe-pointer=all -fno-rounding-math -mconstructor-aliases -target-cpu i686 -target-feature +ssse3 -tune-cpu generic -debugger-tuning=gdb -v -fcoverage-compilation-dir=/data/data/com.termux/files/home -resource-dir /data/data/com.termux/files/usr/lib/clang/13.0.0 -isysroot /data/data/com.termux/files -internal-isystem /data/data/com.termux/files/usr/lib/clang/13.0.0/include -internal-isystem /data/data/com.termux/files/usr/local/include -internal-externc-isystem /data/data/com.termux/files/usr/include/i686-linux-android -internal-externc-isystem /data/data/com.termux/files/include -internal-externc-isystem /data/data/com.termux/files/usr/include -fdebug-compilation-dir=/data/data/com.termux/files/home -ferror-limit 19 -fgnuc-version=4.2.1 -fcolor-diagnostics -o /data/data/com.termux/files/usr/tmp/hello-555716.o -x c hello.c
clang -cc1 version 13.0.0 based upon LLVM 13.0.0 default target i686-unknown-linux-android24
ignoring nonexistent directory "/data/data/com.termux/files/usr/local/include"
ignoring nonexistent directory "/data/data/com.termux/files/include"
#include "..." search starts here:
#include <...> search starts here:
 /data/data/com.termux/files/usr/lib/clang/13.0.0/include
 /data/data/com.termux/files/usr/include/i686-linux-android
 /data/data/com.termux/files/usr/include
End of search list.
 "/data/data/com.termux/files/usr/bin/ld.lld" --sysroot=/data/data/com.termux/files -pie -z noexecstack --warn-shared-textrel -z now -z relro --hash-style=gnu --enable-new-dtags -rpath=/data/data/com.termux/files/usr/lib --eh-frame-hdr -m elf_i386 -dynamic-linker /system/bin/linker -o hello /data/data/com.termux/files/usr/lib/crtbegin_dynamic.o -L/data/data/com.termux/files/usr/lib -L/system/lib /data/data/com.termux/files/usr/tmp/hello-555716.o /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl -lc /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl /data/data/com.termux/files/usr/lib/crtend_android.o
~ $ clang++ hello.cpp -o hello -v
clang version 13.0.0
Target: i686-unknown-linux-android24
Thread model: posix
InstalledDir: /data/data/com.termux/files/usr/bin
 "/data/data/com.termux/files/usr/bin/clang-13" -cc1 -triple i686-unknown-linux-android24 -emit-obj -mrelax-all --mrelax-relocations -mnoexecstack -disable-free -disable-llvm-verifier -discard-value-names -main-file-name hello.cpp -mrelocation-model pic -pic-level 2 -pic-is-pie -mframe-pointer=all -fno-rounding-math -mconstructor-aliases -target-cpu i686 -target-feature +ssse3 -tune-cpu generic -debugger-tuning=gdb -v -fcoverage-compilation-dir=/data/data/com.termux/files/home -resource-dir /data/data/com.termux/files/usr/lib/clang/13.0.0 -isysroot /data/data/com.termux/files -internal-isystem /data/data/com.termux/files/usr/include/c++/v1 -internal-isystem /data/data/com.termux/files/usr/lib/clang/13.0.0/include -internal-isystem /data/data/com.termux/files/usr/local/include -internal-externc-isystem /data/data/com.termux/files/usr/include/i686-linux-android -internal-externc-isystem /data/data/com.termux/files/include -internal-externc-isystem /data/data/com.termux/files/usr/include -fdeprecated-macro -fdebug-compilation-dir=/data/data/com.termux/files/home -ferror-limit 19 -fgnuc-version=4.2.1 -fcxx-exceptions -fexceptions -fcolor-diagnostics -D__GCC_HAVE_DWARF2_CFI_ASM=1 -o /data/data/com.termux/files/usr/tmp/hello-f94420.o -x c++ hello.cpp
clang -cc1 version 13.0.0 based upon LLVM 13.0.0 default target i686-unknown-linux-android24
ignoring nonexistent directory "/data/data/com.termux/files/usr/local/include"
ignoring nonexistent directory "/data/data/com.termux/files/include"
#include "..." search starts here:
#include <...> search starts here:
 /data/data/com.termux/files/usr/include/c++/v1
 /data/data/com.termux/files/usr/lib/clang/13.0.0/include
 /data/data/com.termux/files/usr/include/i686-linux-android
 /data/data/com.termux/files/usr/include
End of search list.
 "/data/data/com.termux/files/usr/bin/ld.lld" --sysroot=/data/data/com.termux/files -pie -z noexecstack --warn-shared-textrel -z now -z relro --hash-style=gnu --enable-new-dtags -rpath=/data/data/com.termux/files/usr/lib --eh-frame-hdr -m elf_i386 -dynamic-linker /system/bin/linker -o hello /data/data/com.termux/files/usr/lib/crtbegin_dynamic.o -L/data/data/com.termux/files/usr/lib -L/system/lib /data/data/com.termux/files/usr/tmp/hello-f94420.o -lc++_shared /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl -lm -lc /data/data/com.termux/files/usr/lib/clang/13.0.0/lib/android/libclang_rt.builtins-i686-android.a -l:libunwind.a -ldl /data/data/com.termux/files/usr/lib/crtend_android.o
```
